ui/src/hooks/useTerminal.ts

```diff
@@ -120,7 +120,7 @@ export const useTerminal = async (el: HTMLElement, option: ICallbackOptions): Pr
       }
       case 'CLOSE': {
         socket.close();
-        sendEventToLuna('CLOSE', '');
+        sendEventToLuna('CLOSE', '',lunaId.value, origin.value);
         break;
       }
       case 'PING':
```

Comment:
There is no major difference between the current version and any earlier versions of this code snippet provided. The key points to consider during review include:

  * This function imports necessary libraries for WebSocket operations, making it more efficient.

  * It sets up initial state variables like `socket`, which will handle all incoming data from server via WebSocket connection using the specified option.

Therefore, no further changes can be recommended as per these details.